internal/venice-common/src/main/java/com/linkedin/venice/meta/Version.java

```diff
@@ -36,6 +37,14 @@ public interface Version extends Comparable<Version>, DataModelBackedStructure<S
    */
   String VENICE_RE_PUSH_PUSH_ID_PREFIX = "venice_re_push_";
 
+  static String getRealTimeTopicNameIfEmpty(String realTimeTopicName, String storeName) {
+    if (Strings.isBlank(realTimeTopicName)) {
+      return composeRealTimeTopic(storeName);
+    } else {
+      return realTimeTopicName;
+    }
```

Comment:
Nit, this function can actually be simplified to a one line statement if we use ternary operands (?:). I don't know if it worthy to define a new function with 2 parameters.

Reply:
I also personally do not like one line methods, but two things made me do this, a) it is used at many places, b) it helps in emphasizing on the fact that RT topic name is strongly related to store name.